exam/views.py

```
@@ -0,0 +1,129 @@
+                a) <Option 1>
+                b) <Option 2>
+                c) <Option 3>
+                d) <Option 4>
+                Answer: <Correct Option>
+                """
+                
+                try:
+                    chat_completion = client.chat.completions.create(
+                        messages=[
+                            {"role": "system", "content": "You are an AI that generates MCQ questions."},
+                            {"role": "user", "content": prompt}
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)